icu4c/source/i18n/number_output.cpp

```diff
@@ -39,6 +39,48 @@ MeasureUnit FormattedNumber::getOutputUnit(UErrorCode& status) const {
     return fData->outputUnit;
 }
 
+NounClass FormattedNumber::getNounClass(UErrorCode &status) const {
+    UPRV_FORMATTED_VALUE_METHOD_GUARD(NounClass::NEUTER);
+    const char *nounClass = fData->gender;
+
+    // if it is not exist, return `NEUTER`
```

Comment:
In this case, we should not return NEUTER.  I feel we should return `EMPTY` or `NOT_SUPPORTED`. what do you think?

Reply:
Hmmm. For plural classes we don't have a special value for (unknown/unspecified/undetermined). The 'other' class always means 'none of the above'. 

That does have the advantage of robustness; if we make changes to introduce a new category of plurals, it is 'carved' it out of 'other', and old messages work the same. So there would be a certain advantage to introducing OTHER here, I think as the first item.

Reply:
Hmm. I think it's really a linguistics question. Is it correct in all known languages that NEUTER is the default noun case?

Reply:
Okay, shall I introduce `OTHER` and update the proposal? I think `OTHER` is a good solution.

Reply:
@sffc , I think `NEUTER` is a category and has its own rules such as common and personal. therefore, I prefer using `OTHER`.

Also, the users should differentiate between the the noun class `NEUTER` and if the language is not supported.

Reply:
@macchiati , I think `OTHER` could be at the end and leave `NEUTER` = 0. what do you think?

Reply:
Okay, I have moved it to the start.

Reply:
The comment has a typo: it says NEUTER but it should say OTHER

Reply:
@sffc , done!